unison-cli/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -963,17 +964,20 @@ notifyUser dir o = case o of
           ( P.syntaxToColor . prettyHashQualified' . fmap Name.fromSegment $ hq,
             isBuiltin r
           )
-        ShallowBranchEntry ns _ count ->
+        ShallowBranchEntry ns _ (NamespaceStats {numContainedTerms, numContainedTypes}) ->
           ( (P.syntaxToColor . prettyName . Name.fromSegment) ns <> "/",
-            case count of
-              Nothing -> "(namespace)"
-              Just 1 -> P.lit "(1 definition)"
-              Just n -> P.lit "(" <> P.shown n <> P.lit " definitions)"
+            case catMaybes [formatCount "term" numContainedTerms, formatCount "type" numContainedTypes] of
```

Comment:
I split this up now that it's easy to do so, but maybe it's just more visual noise, I don't know.

If people prefer the old "definitions" count rather than the more granular it's trivial to switch